vtsclient/vtsclient_grpc.go

```diff
@@ -195,6 +195,54 @@ func (o *GRPC) GetEARSigningPublicKey(ctx context.Context, in *emptypb.Empty, op
 	return c.GetEARSigningPublicKey(ctx, in, opts...)
 }
 
+func (o *GRPC) GetCoservSigningPublicKey(ctx context.Context, in *emptypb.Empty, opts ...grpc.CallOption) (*proto.PublicKey, error) {
```

Comment:
the name of the API should be
GetCoservVerificationPublicKey OR GetCoservVerificationKey

Reply:
Why? It’s the public key associated with the CoSERV signing key … Note that the naming scheme is consistent with that used for the EAR signing key.